### PR TITLE
General improvements and other stuff

### PR DESCRIPTION
**Pull request by fjug**
Take it!

### CODE REVIEW

File: src/main/java/is/ulrik/SciViewTutorial/HelloWorldPlugin.java [hunk 1]
```diff
@@ -1,5 +1,12 @@
 package is.ulrik.SciViewTutorial;
 
+import org.scijava.Context;
```
Comment (skalarproduktraum) [inline src/main/java/is/ulrik/SciViewTutorial/HelloWorldPlugin.java]: Filthy eclipse user, revert this or be gone!

File: src/main/java/is/ulrik/SciViewTutorial/OverlayPlugin.java [hunk 2]
```diff
@@ -1,5 +1,13 @@
 package is.ulrik.SciViewTutorial;
 
+import org.scijava.InstantiableException;
```
Comment (skalarproduktraum) [inline src/main/java/is/ulrik/SciViewTutorial/OverlayPlugin.java]: OMG! More of that...

**Review by skalarproduktraum — CHANGES_REQUESTED** [review]
Looks good, but please don't change the imports.

File: src/main/java/is/ulrik/SciViewTutorial/HelloWorldPlugin.java (hunk 1 above)
Comment (fjug) [reply src/main/java/is/ulrik/SciViewTutorial/HelloWorldPlugin.java]: I'm so sorry... please except my apologies... 
Your devoted friend,
Florian

File: src/main/java/is/ulrik/SciViewTutorial/OverlayPlugin.java (hunk 2 above)
Comment (fjug) [reply src/main/java/is/ulrik/SciViewTutorial/OverlayPlugin.java]: Thanks!

**Review by skalarproduktraum — APPROVED** [review]
Great PR, thank you!

**Pull request merged**